api/v1alpha1/flowcollector_types.go

```diff
@@ -358,6 +358,18 @@ type FlowCollectorFLP struct {
 	// kafkaConsumerBatchSize indicates to the broker the maximum batch size, in bytes, that the consumer will accept. Ignored when not using Kafka. Default: 10MB.
 	KafkaConsumerBatchSize int `json:"kafkaConsumerBatchSize"`
 
+	//+kubebuilder:validation:Pattern:=^\d+(ns|ms|s|m)?$
+	//+kubebuilder:default:="30s"
+	// +optional
+	// connection update interval is the duration of time to wait between update reports of a connection
+	ConnectionUpdateInterval *string `json:"connectionUpdateInterval,omitempty"`
```

Comment:
pls use `metav1.Duration` here instead of `string` with that u won't need to have the regex validation same to the other new field

Reply:
https://github.com/netobserv/network-observability-operator/pull/252/commits/eb2796d108e620ff160f28c01828425e95973e15 done !